### ISSUE TITLE
Unable to send data to graphite when using --preScript - "ERROR: Error: No data to send to graphite for message"

### ISSUE THREAD

**Issue by primehaxor** [sitespeedio/sitespeed.io#2090]
Hello,

  I'm trying to use the preScript and also store the results inside graphite and I'm receiving this kind of error, here is more information:

Testing Env:

docker-compose.yml
```

version: '3'
services:
    sitespeed.io:
      image: sitespeedio/sitespeed.io
      command: -V
      shm_size: 1g
      volumes:
            - ./sitespeed-result/:/sitespeed.io/sitespeed-result
    grafana:
      image: grafana/grafana:5.1.4
      depends_on:
        - graphite
      links:
        - graphite
      ports:
        - "3000:3000"
      environment:
        - GF_SECURITY_ADMIN_PASSWORD=mypass
        - GF_SECURITY_ADMIN_USER=sitespeedio
        - GF_AUTH_ANONYMOUS_ENABLED=true
        - GF_USERS_ALLOW_SIGN_UP=false
        - GF_USERS_ALLOW_ORG_CREATE=false
      volumes:
        - grafana:/var/lib/grafana
      restart: always
    graphite:
      image: sitespeedio/graphite:1.0.2-2-b
      ports:
        - "2003:2003"
        - "8080:80"
      restart: always
      volumes:
        - whisper:/opt/graphite/storage/whisper
        # In production you should configure/map these to your container
        # Download from https://github.com/sitespeedio/sitespeed.io/tree/master/docker/graphite
        #- ./graphite/graphite.db:/opt/graphite/storage/graphite.db
        #- ./graphite/conf/storage-schemas.conf:/opt/graphite/conf/storage-schemas.conf
        #- ./graphite/conf/storage-aggregation.conf:/opt/graphite/conf/storage-aggregation.conf
        #- ./graphite/conf/carbon.conf:/opt/graphite/conf/carbon.conf
    grafana-setup:
      image: sitespeedio/grafana-bootstrap:6.3
      links:
        - grafana
      environment:
        - GF_PASSWORD=mypass
        - GF_USER=sitespeedio
volumes:
    grafana:
    whisper:
```

```
   module.exports = {
    run(context) {
        return context.runWithDriver((driver) => {
                return driver.get(context.url)
                    .then(() => {
                 const webdriver = context.webdriver;
        const until = webdriver.until;
        const By = webdriver.By;
        driver.manage().addCookie({name:"skippoupup", value:"true"}).then(function(){
            driver.get(context.url);
        });
    });
    });
    }
   };

```
**Firefox Testing**
$ docker run --privileged --shm-size=1g --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io --preScript test.js -b firefox https://www.mysite.com -n 1 --graphite.host 192.168.1.104

```
Google Chrome 67.0.3396.79 
Mozilla Firefox 61.0
[2018-06-30 08:51:13] INFO: Versions OS: linux 4.9.87-linuxkit-aufs nodejs: v8.11.1 sitespeed.io: 7.1.2 browsertime: 3.1.2 coach: 2.0.2
[2018-06-30 08:51:13] INFO: Starting firefox for analysing https://www.mysite.com 1 time(s)
[2018-06-30 08:51:13] INFO: Testing url https://www.mysite.com iteration 1
[2018-06-30 08:51:32] INFO: Waiting on har-export-trigger to collect the HAR
[2018-06-30 08:51:47] INFO: 122 requests, 24.04 kb, backEndTime: 34ms, firstPaint: 854ms, firstVisualChange: 867ms, DOMContentLoaded: 884ms, Load: 1.98s, speedIndex: 979, visualComplete85: 1.07s, lastVisualChange: 1.63s, rumSpeedIndex: 795
[2018-06-30 08:51:47] ERROR: Error: No data to send to graphite for message:
{
  "uuid": "c21aef66-dff5-480f-8195-1b78deddf7fa",
  "type": "largestassets.summary",
  "timestamp": "2018-06-30T08:51:47+00:00",
  "source": "assets",
  "data": {
    "image": [
      {
        "transferSize": null
      }
    ]
  },
  "group": "www.mysite.com"
}
    at Object.processMessage (/usr/src/app/lib/plugins/graphite/index.js:103:9)
    at queue.process.message (/usr/src/app/lib/core/queueHandler.js:208:32)
From previous event:
    at queue.process.message (/usr/src/app/lib/core/queueHandler.js:208:17)
    at drainItem (/usr/src/app/node_modules/concurrent-queue/index.js:96:21)
    at Immediate.drain [as _onImmediate] (/usr/src/app/node_modules/concurrent-queue/index.js:77:84)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
[2018-06-30 08:51:48] INFO: HTML stored in /sitespeed.io/sitespeed-result/www.mysite.com/2018-06-30-08-51-13
[2018-06-30 08:51:48] INFO: Finished analysing https://www.mysite.com
```

**Chrome Testing**
$ docker run --privileged --shm-size=1g --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io --preScript test.js -b chrome https://www.mysite.com -n 1 --graphite.host 192.168.1.104

```
Google Chrome 67.0.3396.79 
Mozilla Firefox 61.0
[2018-06-30 08:54:52] INFO: Versions OS: linux 4.9.87-linuxkit-aufs nodejs: v8.11.1 sitespeed.io: 7.1.2 browsertime: 3.1.2 coach: 2.0.2
[2018-06-30 08:54:53] INFO: Starting chrome for analysing https://www.mysite.com 1 time(s)
[2018-06-30 08:54:53] INFO: Testing url https://www.mysite.com iteration 1
[2018-06-30 08:55:17] INFO: 62 requests, 42.75 kb, backEndTime: 5ms, firstPaint: 438ms, firstVisualChange: 434ms, DOMContentLoaded: 405ms, Load: 1.49s, speedIndex: 543, visualComplete85: 634ms, lastVisualChange: 1s, rumSpeedIndex: 453
[2018-06-30 08:55:17] ERROR: Error: Failed to calculate min for stats: {
  "_config": {
    "store_data": true
  },
  "data": [
    null,
    163.845,
    null,
    null,
    null,
    null,
    null
  ],
  "length": 7,
  "sum": null,
  "sum_of_squares": null,
  "sum_of_logs": null,
  "sum_of_square_of_logs": null,
  "zeroes": 0,
  "min": null,
  "max": null,
  "_stddev": null,
  "_data_sorted": [
    null,
    163.845,
    null,
    null,
    null,
    null,
    null
  ]
}
    at percentiles.forEach.p (/usr/src/app/lib/support/statsHelpers.js:64:15)
    at Array.forEach (<anonymous>)
    at Object.summarizeStats (/usr/src/app/lib/support/statsHelpers.js:58:17)
    at reduce (/usr/src/app/lib/plugins/domains/aggregator.js:48:34)
    at /usr/src/app/node_modules/lodash.reduce/index.js:202:9
    at /usr/src/app/node_modules/lodash.reduce/index.js:1284:11
    at baseForOwn (/usr/src/app/node_modules/lodash.reduce/index.js:903:20)
    at /usr/src/app/node_modules/lodash.reduce/index.js:1253:14
    at baseReduce (/usr/src/app/node_modules/lodash.reduce/index.js:199:3)
    at reduce (/usr/src/app/node_modules/lodash.reduce/index.js:1842:10)
    at calc (/usr/src/app/lib/plugins/domains/aggregator.js:40:10)
    at Object.summarize (/usr/src/app/lib/plugins/domains/aggregator.js:118:16)
    at Object.processMessage (/usr/src/app/lib/plugins/domains/index.js:34:36)
    at queue.process.message (/usr/src/app/lib/core/queueHandler.js:208:32)
    at drainItem (/usr/src/app/node_modules/concurrent-queue/index.js:96:21)
    at Immediate.drain (/usr/src/app/node_modules/concurrent-queue/index.js:77:84)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
[2018-06-30 08:55:18] INFO: HTML stored in /sitespeed.io/sitespeed-result/www.mysite.com/2018-06-30-08-54-52
[2018-06-30 08:55:18] INFO: Finished analysing https://www.mysite.com

```
**Testing without graphite**
$ docker run --privileged --shm-size=1g --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io --preScript test.js -b firefox https://www.mysite.com -n 1                               
```
Google Chrome 67.0.3396.79 
Mozilla Firefox 61.0
[2018-06-30 09:01:15] INFO: Versions OS: linux 4.9.87-linuxkit-aufs nodejs: v8.11.1 sitespeed.io: 7.1.2 browsertime: 3.1.2 coach: 2.0.2
[2018-06-30 09:01:15] INFO: Starting firefox for analysing https://www.mysite.com 1 time(s)
[2018-06-30 09:01:15] INFO: Testing url https://www.mysite.com iteration 1
[2018-06-30 09:01:33] INFO: Waiting on har-export-trigger to collect the HAR
[2018-06-30 09:01:45] INFO: 123 requests, 38.41 kb, backEndTime: 43ms, firstPaint: 1.03s, firstVisualChange: 1.03s, DOMContentLoaded: 1.08s, Load: 2.82s, speedIndex: 1110, visualComplete85: 1.17s, lastVisualChange: 1.80s, rumSpeedIndex: 976
[2018-06-30 09:01:46] INFO: HTML stored in /sitespeed.io/sitespeed-result/www.mysite.com/2018-06-30-09-01-15
[2018-06-30 09:01:46] INFO: Finished analysing https://www.mysite.com
```

$ docker run --privileged --shm-size=1g --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io --preScript test.js -b chrome https://www.mysite.com -n 1 
```
Google Chrome 67.0.3396.79 
Mozilla Firefox 61.0
[2018-06-30 09:01:57] INFO: Versions OS: linux 4.9.87-linuxkit-aufs nodejs: v8.11.1 sitespeed.io: 7.1.2 browsertime: 3.1.2 coach: 2.0.2
[2018-06-30 09:01:58] INFO: Starting chrome for analysing https://www.mysite.com 1 time(s)
[2018-06-30 09:01:58] INFO: Testing url https://www.mysite.com iteration 1
[2018-06-30 09:02:24] INFO: 60 requests, 22.27 kb, backEndTime: 7ms, firstPaint: 395ms, firstVisualChange: 434ms, DOMContentLoaded: 599ms, Load: 1.96s, speedIndex: 706, visualComplete85: 900ms, lastVisualChange: 2.67s, rumSpeedIndex: 707
[2018-06-30 09:02:24] ERROR: Error: Failed to calculate min for stats: {
  "_config": {
    "store_data": true
  },
  "data": [
    null,
    173.826,
    null,
    null,
    null,
    null,
    null
  ],
  "length": 7,
  "sum": null,
  "sum_of_squares": null,
  "sum_of_logs": null,
  "sum_of_square_of_logs": null,
  "zeroes": 0,
  "min": null,
  "max": null,
  "_stddev": null,
  "_data_sorted": [
    null,
    173.826,
    null,
    null,
    null,
    null,
    null
  ]
}
    at percentiles.forEach.p (/usr/src/app/lib/support/statsHelpers.js:64:15)
    at Array.forEach (<anonymous>)
    at Object.summarizeStats (/usr/src/app/lib/support/statsHelpers.js:58:17)
    at reduce (/usr/src/app/lib/plugins/domains/aggregator.js:48:34)
    at /usr/src/app/node_modules/lodash.reduce/index.js:202:9
    at /usr/src/app/node_modules/lodash.reduce/index.js:1284:11
    at baseForOwn (/usr/src/app/node_modules/lodash.reduce/index.js:903:20)
    at /usr/src/app/node_modules/lodash.reduce/index.js:1253:14
    at baseReduce (/usr/src/app/node_modules/lodash.reduce/index.js:199:3)
    at reduce (/usr/src/app/node_modules/lodash.reduce/index.js:1842:10)
    at calc (/usr/src/app/lib/plugins/domains/aggregator.js:40:10)
    at Object.summarize (/usr/src/app/lib/plugins/domains/aggregator.js:118:16)
    at Object.processMessage (/usr/src/app/lib/plugins/domains/index.js:34:36)
    at queue.process.message (/usr/src/app/lib/core/queueHandler.js:208:32)
    at drainItem (/usr/src/app/node_modules/concurrent-queue/index.js:96:21)
    at Immediate.drain (/usr/src/app/node_modules/concurrent-queue/index.js:77:84)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
[2018-06-30 09:02:25] INFO: HTML stored in /sitespeed.io/sitespeed-result/www.mysite.com/2018-06-30-09-01-57
[2018-06-30 09:02:25] INFO: Finished analysing https://www.mysite.com
```

**Testing Chrome/Firefox without --preScript**
$ docker run --privileged --shm-size=1g --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io -b chrome https://www.mysite.com -n 1 --graphite.host 192.168.1.104 

```
Google Chrome 67.0.3396.79 
Mozilla Firefox 61.0
[2018-06-30 08:57:11] INFO: Versions OS: linux 4.9.87-linuxkit-aufs nodejs: v8.11.1 sitespeed.io: 7.1.2 browsertime: 3.1.2 coach: 2.0.2
[2018-06-30 08:57:11] INFO: Starting chrome for analysing https://www.mysite.com 1 time(s)
[2018-06-30 08:57:11] INFO: Testing url https://www.mysite.com iteration 1
[2018-06-30 08:57:26] INFO: 91 requests, 2948.01 kb, backEndTime: 140ms, firstPaint: 796ms, firstVisualChange: 800ms, DOMContentLoaded: 1.88s, Load: 2.13s, speedIndex: 1907, visualComplete85: 2.40s, lastVisualChange: 2.73s, rumSpeedIndex: 1962
[2018-06-30 08:57:28] INFO: HTML stored in /sitespeed.io/sitespeed-result/www.mysite.com/2018-06-30-08-57-11
[2018-06-30 08:57:28] INFO: Finished analysing https://www.mysite.com

```

$ docker run --privileged --shm-size=1g --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io -b firefox https://www.mysite.com -n 1 --graphite.host 192.168.1.104

```
Google Chrome 67.0.3396.79 
Mozilla Firefox 61.0
[2018-06-30 08:57:37] INFO: Versions OS: linux 4.9.87-linuxkit-aufs nodejs: v8.11.1 sitespeed.io: 7.1.2 browsertime: 3.1.2 coach: 2.0.2
[2018-06-30 08:57:37] INFO: Starting firefox for analysing https://www.mysite.com 1 time(s)
[2018-06-30 08:57:37] INFO: Testing url https://www.mysite.com iteration 1
[2018-06-30 08:57:48] INFO: Waiting on har-export-trigger to collect the HAR
[2018-06-30 08:57:58] INFO: 97 requests, 3013.94 kb, backEndTime: 229ms, firstPaint: 868ms, firstVisualChange: 933ms, DOMContentLoaded: 1.41s, Load: 2.41s, speedIndex: 1633, visualComplete85: 2.10s, lastVisualChange: 2.47s, rumSpeedIndex: 1487
[2018-06-30 08:58:00] INFO: HTML stored in /sitespeed.io/sitespeed-result/www.mysite.com/2018-06-30-08-57-37
[2018-06-30 08:58:00] INFO: Finished analysing https://www.mysite.com

```

**Comment by primehaxor** [comment]
Just an update, I can see the data inside the graphana dashboard, comparing the results with details.html looks like it is valid data.

**Comment by soulgalore** [comment]
Hi @primehaxor and thanks for the detailed bug report!

It seems that we don't handle everything correctly, lets see when we can fix that we just moved into vacation mode: https://github.com/sitespeedio/sitespeed.io#vacation-july---august

One thing you can do  I think is to set the cookie directly with request header, then you will not need to access the URL twice (where you set the cookie first) if I understand correctly? Can you try that?

You can add the header with:
`  --browsertime.requestheader` Request header that will be added to the request. Add multiple  instances to add multiple request headers. Use the following formatkey:value

If we can make that work, that is also good for the metrics, since your pretask will cache assets in the browser cache?

But we will probably still have the other bug .... hmm can you email me the URL (if its public) and I can do a quick check tonight. If I'm able to reproduce it's probably easy to fix. peter [AT] soulgalore [DOT] com

Best
Peter

**Comment by soulgalore** [comment]
Cool I will have a look. The cookie trick I said didn't work, we have PR to fix it in https://github.com/sitespeedio/browsertime-extension/pull/4 but I haven't got around to test, but when I got a good URL, I can do it. I could reproduce your errors so I will try more tonight.

Best
Peter

**Comment by soulgalore** [comment]
Hm, I fixed the extension so it's now easy to add one cookie like --cookie name=value and it works fine in Chrome but not Firefox ...  I'll probably release it anyhow as Chrome only as it will help, and then I also fix that other. That should make it much easier to just remove that first screen.

**Comment by primehaxor** [comment]
Alright, thank you @soulgalore for the update.

**Comment by soulgalore** [comment]
@primehaxor  I pushed 7.2.0 just now that hopefully fix your problem. Remove your preScript and add the cookie like this: `--cookie name=value`

Let me know if it fixes the problem!
Best
Peter

**Comment by primehaxor** [comment]
@soulgalore I ran about eight times with the setup ( firefox, chrome ) ( 3g, 3gfast, 3gslow, cable ) also I recorded a video to make sure that I won't see the poup-up anymore, so far it is working perfectly, i just had one issue but i think it isn't related i've paste in the end. Thank you!!


$ docker run --privileged  --shm-size=1g --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io:latest --cookie "skippoupup=true" --video -b chrome https://www.mysite.com -n 1 --graphite.host 192.168.1.104 --browsertime.viewPort 1920x1200 -c $CONNECTION

```
Google Chrome 67.0.3396.99 
Mozilla Firefox 61.0
[2018-07-01 07:15:24] INFO: Versions OS: linux 4.9.87-linuxkit-aufs nodejs: v8.11.1 sitespeed.io: 7.2.0 browsertime: 3.2.0 coach: 2.0.5
[2018-07-01 07:15:24] INFO: Starting chrome for analysing https://www.mysite.com 1 time(s)
[2018-07-01 07:15:24] INFO: Testing url https://www.mysite.com iteration 1
[2018-07-01 07:15:45] INFO: 146 requests, 3115.63 kb, backEndTime: 220ms, firstPaint: 972ms, firstVisualChange: 1s, DOMContentLoaded: 1.55s, Load: 3.57s, speedIndex: 1552, visualComplete85: 1.93s, lastVisualChange: 4.17s, rumSpeedIndex: 1828
[2018-07-01 07:15:47] INFO: HTML stored in /sitespeed.io/sitespeed-result/www.mysite.com/2018-07-01-07-15-24
[2018-07-01 07:15:47] INFO: Finished analysing https://www.mysite.com
```


$ docker run --privileged  --shm-size=1g --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io:latest --cookie "skippoupup=true" --video -b firefox https://www.mysite.com -n 1 --graphite.host 192.168.1.104 --browsertime.viewPort 1920x1200 -c $CONNECTION

```
Google Chrome 67.0.3396.99 
Mozilla Firefox 61.0
[2018-07-01 07:17:24] INFO: Versions OS: linux 4.9.87-linuxkit-aufs nodejs: v8.11.1 sitespeed.io: 7.2.0 browsertime: 3.2.0 coach: 2.0.5
[2018-07-01 07:17:24] INFO: Starting firefox for analysing https://www.mysite.com 1 time(s)
[2018-07-01 07:17:25] INFO: Testing url https://www.mysite.com iteration 1
[2018-07-01 07:17:46] INFO: Waiting on har-export-trigger to collect the HAR
[2018-07-01 07:18:00] INFO: 147 requests, 3197.13 kb, backEndTime: 372ms, firstPaint: 1.75s, firstVisualChange: 1.80s, DOMContentLoaded: 1.85s, Load: 8.65s, speedIndex: 2102, visualComplete85: 2.33s, lastVisualChange: 2.93s, rumSpeedIndex: 1875
[2018-07-01 07:18:02] INFO: HTML stored in /sitespeed.io/sitespeed-result/www.mysite.com/2018-07-01-07-17-24
[2018-07-01 07:18:02] INFO: Finished analysing https://www.mysite.com 
```

During one test i just received it i don't know it is related

```
peedIndex: 2827 (±232.35), visualComplete85: 2.71s (±151.13ms), lastVisualChange: 11.59s (±2.76s), rumSpeedIndex: 1714 (±35.09) (3 runs)
[2018-07-01 08:15:18] ERROR: Error: Tried to add NaN to stats for path contentSize
    at Object.pushStats (/usr/src/app/lib/support/statsHelpers.js:27:13)
    at Object.pushGroupStats (/usr/src/app/lib/support/statsHelpers.js:35:10)
    at /usr/src/app/lib/plugins/pagexray/pagexrayAggregator.js:22:22
    at Array.forEach (<anonymous>)
    at /usr/src/app/lib/plugins/pagexray/pagexrayAggregator.js:21:20
    at Array.forEach (<anonymous>)
    at Object.addToAggregate (/usr/src/app/lib/plugins/pagexray/pagexrayAggregator.js:19:17)
    at Object.processMessage (/usr/src/app/lib/plugins/pagexray/index.js:76:30)
    at queue.process.message (/usr/src/app/lib/core/queueHandler.js:208:32)
    at drainItem (/usr/src/app/node_modules/concurrent-queue/index.js:96:21)
    at Immediate.drain (/usr/src/app/node_modules/concurrent-queue/index.js:77:84)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
[2018-07-01 08:15:18] ERROR: TypeError: Cannot read property 'image' of undefined
    at module.exports (/usr/src/app/lib/plugins/html/setup/summaryBoxes.js:111:28)
    at HTMLBuilder.render (/usr/src/app/lib/plugins/html/htmlBuilder.js:73:26)
    at Object.processMessage (/usr/src/app/lib/plugins/html/index.js:199:35)
    at queue.process.message (/usr/src/app/lib/core/queueHandler.js:208:32)
    at drainItem (/usr/src/app/node_modules/concurrent-queue/index.js:96:21)
    at Immediate.drain (/usr/src/app/node_modules/concurrent-queue/index.js:77:84)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)

```

**Comment by soulgalore** [comment]
Hi @primehaxor hmm the first one it seems that Firefox doesn't create a valid har (`Tried to add NaN to stats for path contentSize` if I understand correctly). And the other error is on our side (`Cannot read property 'image' of undefined`). Is that the same URL that you sent me yesterday? If that's the case let me check again tonight, else if you can send me the URL.

Best
Peter

**Comment by primehaxor** [comment]
Hello @soulgalore yes it is the same one, but there is no pattern to happen, after this error, I've tried to run again and it didn't happen.

**Comment by soulgalore** [comment]
There's some content (I guess) but not 100%, the problematic HAR is annoying. I'll be away the coming week but maybe @beenanner or @tobli can have a look if they have time (I shared the URL privately to them).

**Comment by tobli** [comment]
This is interesting, it exposes a few flaws in error handling on unexpected/missing content. Sitespeed.io should not spew errors in the log like that, regardless of other issues. An opportunity for some hardening.

**Comment by soulgalore** [comment]
@primehaxor is it possible for you to share the HAR file from a failing run? that will help us to fix the problem in pagexray.

Best
Peter

**Comment by primehaxor** [comment]
@soulgalore sure I will provide it to you.

**Comment by primehaxor** [comment]
@soulgalore just one update I noticed during the error time the website wasn't working ( it was "waiting for connection" for more than 60 seconds )

**Comment by soulgalore** [comment]
thanks @primehaxor yeah but we should take care of the broken HAR anyway, thanks for providing them!

**Comment by primehaxor** [comment]
@soulgalore wy, btw after many tests I was able to see this problem is only happening with firefox, on the chrome side no errors atm.

**Comment by soulgalore** [comment]
@primehaxor yes that would be super, thanks!

**Comment by soulgalore** [comment]
One more thing: I looked at the HAR and I'm pretty sure this is an upstream issue in Firefox/https://github.com/devtools-html/har-export-trigger (that generates a faulty HAR), so we can reference the new issue when we create the upstream one. I'll do a fix in PageXray (that's create the error on our side) so we handle the broken HAR.
